pytests/src/buf_and_str.rs

```diff
@@ -43,7 +43,7 @@ impl BytesExtractor {
 
 #[pyfunction]
 fn return_memoryview(py: Python<'_>) -> PyResult<&PyMemoryView> {
-    let bytes: &PyAny = PyBytes::new(py, b"hello world").into();
+    let bytes: &PyAny = PyBytes::new_bound(py, b"hello world").into_gil_ref();
```

Comment:
after #3786 this could return a `Bound<PyMemoryView>` now 👼

Reply:
Ah, lovely! 🚀